src/components/collection-list/collection-list.tsx

```diff
@@ -28,6 +27,7 @@ interface IProps {
   showControls?: boolean;
   handleControlClick?: (id, event) => void;
   repo?: string;
+  renderCollectionControls: (collection) => React.ReactElement;
```

Comment:
ah, better, thanks :)

Though you may want to prefer ReactNode over ReactElement...

https://stackoverflow.com/a/58123882/535728
> A ReactElement is an object with a type and props.
> A ReactNode is a ReactElement, a ReactFragment, a string, a number or an array of ReactNodes, or null, or undefined, or a boolean

```suggestion
  renderCollectionControls: (collection) => React.ReactNode;
```